src/App.tsx

```diff
@@ -13,15 +14,16 @@ export const App = (): React.ReactElement => {
 	const [ isLoggedIn, setIsLoggedIn ] = useState<boolean>(false)
 	const [ roller, setRoller ] = useState<string[]>([])
 
-	const harTilgangTilArrangor = roller.filter((rolle) => rolle === 'KOORDINATOR')
-		.length > 0
-
 	useEffect(() => {
 		if (isResolved(fetchMineRollerPromise)) {
 			setRoller(fetchMineRollerPromise.result.data)
 			setIsLoggedIn(true)
 		}
 
+		if (!roller.includes('KOORDINATOR') && roller.includes('VEILEDER')) {
```

Comment:
Dette er også en feature toggle? Slik at de som har begge rollene kan fortsette som vanlig men ikke de som kun er veiledere? Ville veldig gjerne likt å lagt denne også inn i en samleklasse for toggles.

Reply:
Nei, dette er ikke en featuretoggle :) En featuretoggle er jo noe som skal skru funksjonalitet av eller på - dette er en varig sjekk som sier at hvis du kun er veileder så skal du redirectes til veileder-oversikten (men nå redirecter den til ingen rolle siden det er sånn det skal oppføre seg i dag).

Reply:
Aha skjønner 😄